liota/lib/utilities/identity.py

```diff
@@ -29,3 +29,28 @@
 #  ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF     #
 #  THE POSSIBILITY OF SUCH DAMAGE.                                            #
 # ----------------------------------------------------------------------------#
+
+import logging
+
+log = logging.getLogger(__name__)
+
+
+class Identity:
+    """
+    This class encapsulates certificates and credentials related to a connection both at Dcc and Device side.
+    """
+
+    def __init__(self, root_ca_cert, username, password, cert_file, key_file):
+        """
+        :param root_ca_cert: Root CA certificate path or Self-signed server certificate path
+        :param username: Username
+        :param password: Corresponding password
+        :param cert_file: Device certificate file path
+        :param key_file: Device certificate key-file path
+        """
+        self.root_ca_cert = root_ca_cert
+        self.username = username
+        self.password = password
+        self.cert_file = cert_file
+        self.key_file = key_file
+        log.debug("Created Identity.")
```

Comment:
For debugging, these kind of logs are not very useful. It is better to provide more information in debug logs like with what username the identity is created etc.